forc/src/ops/forc_init.rs

```diff
@@ -85,13 +85,30 @@ pub fn init(command: InitCommand) -> Result<()> {
 
     match command.template {
         Some(template) => {
+            let example_url =
+                format!("https://github.com/FuelLabs/sway/tree/{SWAY_GIT_TAG}/examples/{template}");
             let template_url = match template.as_str() {
-                "counter" => {
-                    Url::parse("https://github.com/FuelLabs/sway/tree/master/examples/hello_world")?
-                }
+                "counter" => Url::parse(&example_url)?,
+                "fizzbuzz" => Url::parse(&example_url)?,
+                "liquidity_pool" => Url::parse(&example_url)?,
+                "msg_sender" => Url::parse(&example_url)?,
+                "native_token" => Url::parse(&example_url)?,
+                "storage_example" => Url::parse(&example_url)?,
+                "subcurrency" => Url::parse(&example_url)?,
+                "wallet_smart_contract" => Url::parse(&example_url)?,
+                "asm_return_tuple_pointer" => Url::parse(&example_url)?,
```

Comment:
To avoid future headaches, alphabetize.

Reply:
To keep the branching lower, this could also be written as:

```rust
                "counter"
                | "fizzbuzz"
                | "liquidity_pool"
                | "msg_sender"
                | "native_token"
                | "storage_example"
                | "subcurrency"
                | "wallet_smart_contract"
                | "asm_return_tuple_pointer" => Url::parse(&example_url)?,
```